lib/pipefy_message/logger.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require "English"
+require "json"
+require "logger"
+
+module PipefyMessage
+  # Custom Logger class to centralize and deal with logs needs
+  class CustomLogger
```

Comment:
@jleber can't we just set the log level in the app configuration to an env variable? 
https://guides.rubyonrails.org/debugging_rails_applications.html#log-levels

Reply:
Yes, we can but in this way, we'll change all pipefy-core log level 😅 .
The idea here is to change only the async logs without effect on the main application, this was aligned with the SRE team due to the high costs of logs at DataDog (debug level is not indexed/ingested).

It's worth mentioning that this approach is applied only to the log that has a level different from "ERROR".